templates/download/server/thank-you.html

```diff
@@ -32,71 +32,24 @@ <h1>Thank you for downloading Ubuntu Server {{ version }}{{ " LTS" if lts }}</h1
         </p>
       </div>
       <div class="col">
-        {% with returnUrl="/download/server", ga_event_title="Ubuntu Server download" %}
-          {% include 'shared/_download-newsletter.html' %}
-        {% endwith %}
-      </div>
-    </div>
-  </section>
-
-  {% include 'download/shared/_server_weekly_news.html' %}
-
-  <section class="p-section">
-    <hr class="p-rule is-fixed-width" />
-    <div class="row--50-50">
-      <div class="col">
+        {% include 'download/shared/_server_weekly_news.html' %}
         <h2>Ubuntu CLI cheatsheet</h2>
```

Comment:
```suggestion
        <h2>Get Ubuntu Server pro tips</h2>
```

Reply:
I believe this heading should be changed, as per copydoc.

Reply:
Hi @muhammad-ali-pk, looks like the copydoc wasn't as up to date as the web page, we moved towards "CLI cheatsheet" — I made the adjustment and added a comment on the copydoc.